tinkerbell/stack/templates/kubevip.yaml

```diff
@@ -27,6 +27,10 @@ spec:
           value: "true"
         - name: enableServicesElection
           value: "true"
+        {{- with .Values.stack.kubevip.prometheus }}
+        - name: prometheus_server
+          value: {{ .addr }}:{{ .port }}
+        {{- end }}
```

Comment:
We should open up env vars generically here. This way we dont need to keep up to date on all possible options as the come and go. We do this in Smee: https://github.com/tinkerbell/charts/blob/e34a6406b068ae7ee305a2137f592b9e732f4849/tinkerbell/smee/templates/deployment.yaml#L67
You don't have to follow this naming convention (`additionalEnv`) though. It could just something like `.Values.env` or similar.

Reply:
Yeah that sounds like a great idea. will update the PR shortly!